python/pyspark/sql/session.py

```diff
@@ -287,7 +287,12 @@ def getOrCreate(self) -> "SparkSession":
     _instantiatedSession: ClassVar[Optional["SparkSession"]] = None
     _activeSession: ClassVar[Optional["SparkSession"]] = None
 
-    def __init__(self, sparkContext: SparkContext, jsparkSession: Optional[JavaObject] = None):
+    def __init__(
+        self,
+        sparkContext: SparkContext,
+        jsparkSession: Optional[JavaObject] = None,
+        options: Optional[Dict[str, Any]] = None,
+    ):
```

Comment:
This seems to be not a breaking change in Python, right? How do you think about this, @HyukjinKwon ?

Reply:
Yeah I don't think this is breaking. Let me double check closely by tmr EODbut from a coursory look it seems fine.